test/mv_opentelemetry/oban_test.exs

```diff
@@ -0,0 +1,29 @@
+defmodule MvOpentelemetry.ObanTest do
+  use MvOpentelemetry.OpenTelemetryCase
+
+  test "perform_job/1 emits span events" do
```

Comment:
I can't seem to be able to test the ones that return errors, so we will have to trust that it works.